chapters/en/slides/module5/module5_05.Rmd

```diff
@@ -0,0 +1,392 @@
+ .resolve_scale(color='independent'))
+```
+
+Notes:
+The leftmost plot here uses the same default color scheme as in the previous slide.
+
+The plot on the right has a color scheme that adds a slight variation in hue
+while keeping the perceptually uniform variation in lightness.
+
+The reason for this is to highlight small variations in the data
+when this is of great importance. 
+We can see the effect of this buy studying the values in the 60-80 range in both plots.
+It is hard to see the differences between these in the leftmost plot,
+but thanks to the hue transition from yellow to green in the rightmost plot
+we can more clearly perceive detail in this range.
+
+Although the hue shifts in this plots are more gradual
+than in the plot with the categorical color scheme in the previous slide,
+it may still introduce a slight bias towards seeing groupings where there are none,
+which is something to keep in mind.
+
+In practice,
+both of these color schemes are used often to represent quantitative data.
+
+---
+
+## High values should have the most contrast to the background
+
+```{python}
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars()
+alt.Chart(cars).mark_point(size=50, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Horsepower', scale=alt.Scale(scheme='viridis', reverse=True)))
+```
+
+Notes:
+To decide whether the highest or lowest values
+should be at the lightest or darkest end of a sequential color schemes,
+it is advisable to set it so that the most important values
+has the most contrast to the background.
+
+Often there is no difference in importance between high and low values;
+then the convention is to set the highest values
+to have the most contrast with the background,
+which is the case for most Altair color schemes by default.
+
+However,
+for the "viridis" color scheme that we are using here,
+goes from dark to light by default
+so we need to pass `reverse=True` to `alt.Scale`
+(this technique could be used to reverse any scale in Altair,
+including the value along one of the axes).
+
+The viridis color scheme was designed by the developers of matplotlib (another Python plotting package).
+In addition to the color schemes linked in the previous slide deck,
+more details and examples are available in the [matplotlib documentation],
+and many of the core design principles are outlined in [this entertaining talk],
+including the design choice behind viridis.
+
+[matplotlib documentation]: http://matplotlib.org/users/colormaps.html
+[this entertaining talk]: https://www.youtube.com/watch?v=xAoljeRJ3lU
+
+---
+
+## Some color schemes are designed specifically for people with color vision deficiencies
+
+```{python}
+alt.Chart(cars).mark_point(size=50, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Horsepower', scale=alt.Scale(scheme='cividis', reverse=True)))
+```
```

Comment:
The colorbars don't show up on some of these plots, it will hopefully be fixes when we switch to html.

Reply:
fixed by html